### DISCUSSION TITLE
Project Question

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#168] [Q&A] [answered]
As we begin our projects, I think Sean mentioned that we will be using a URL/page within GitHub. Does anyone know how I should go about starting this? Do I create a new repo or something? I'm ready to start coding, but I don't know where to put it.

**Comment by jakegros** [comment, marked as answer]
I believe you should start your GitHub page in your own repo.  Then, add the url to the project link https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/132#discussion-5309248

FYI, I am having issues using the Project template referenced in the link above.

**Reply by Skye404** [reply]
Thanks! The link referenced worked for me.
